src/Route.jsx

```diff
@@ -60,6 +60,22 @@ export function getRoutesLayer(geojson) {
   }));
 }
 
+export function getSubRoutesLayer(subroute) {
+  const subrouteLayer = new PathLayer({
+    id: 'path-layer',
+    data: {
+      ...subroute,
+    },
+    pickable: true,
+    widthScale: 20,
+    widthMinPixels: 2,
+    getPath: d => d.path,
+    getColor: d => d.color,
+    getWidth: 5,
+  });
+  return subrouteLayer;
```

Comment:
@EddyIonescu 
I based this structure off of https://github.com/uber/deck.gl/blob/master/docs/layers/path-layer.md
data looks like it's in the right structure